ql/src/test/org/apache/hadoop/hive/ql/exec/TestGetPartitionAuthWithBatches.java

```diff
@@ -50,6 +54,12 @@
 
 public class TestGetPartitionAuthWithBatches {
 
+    @ClassRule
+    public static HiveTestEnvSetup env_setup = new HiveTestEnvSetup();
+
+    @Rule
+    public TestRule methodRule = env_setup.getMethodRule();
+
```

Comment:
I see these being added at multiple places, any pointers why it is needed in the scope of surefire upgrade?

Reply:
while implementing since [HIVE-27972](https://issues.apache.org/jira/browse/HIVE-27972) I realized that this kind of boilerplate can be avoided by using a well-prepared hive conf, like:
```
new HiveConfForTest(getClass())
```
could this be used here?